src/app/complex/complex-results/complex-navigator-buttons/complex-navigator-buttons.component.css

```diff
@@ -33,6 +32,10 @@
   display: none;
 }
 
+label {
```

Comment:
Why do we need this?

Reply:
We made some modification for the menus so the user can click directly on the whole tile rectangle instead of clicking on the radiobutton, to do so we encapsulated it in the label which needed some style adjustments